Firebase.podspec

```diff
@@ -105,12 +105,6 @@ Simplify your iOS development, grow your user base, and monetize more effectivel
     ss.ios.deployment_target = '9.0'
   end
 
-  s.subspec 'InAppMessagingDisplay' do |ss|
```

Comment:
I'm not sure we want to remove it from the `Firebase.podspec` yet, right? It has only been a single release so far, so if someone missed last release and updates to this one the error won't be nearly as user friendly. Or maybe more accurately, they'll be locked to the previous Firebase version?

Reply:
They're already locked to the currently published deprecated release.  We have not plans to do future releases, so no reason to keep it in the repository.

Reply:
But if someone still has `Firebase/InAppMessaging` in their podfile and we publish a new version of the `Firebase` pod without this subspec, what will happen? Will CocoaPods stay on the newest version that includes that subspec, will it try to get the newest `Firebase` without this subspec and fail, something else?

Reply:
Oh sorry. Now I see your point:
```

[!] CocoaPods could not find compatible versions for pod "Firebase/InAppMessagingDisplay":
  In Podfile:
    Firebase/InAppMessagingDisplay (from `../../../`)

None of your spec sources contain a spec satisfying the dependency: `Firebase/InAppMessagingDisplay (from `../../../`)`.

You have either:
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.
```